### ISSUE TITLE
nomaivr patches pause so the game pauses when any client reads something

### ISSUE THREAD

**Issue by JohnCorby** [qsb-dev/quantum-space-buddies#694]
see forum post [here](https://discord.com/channels/929708786027999262/1290117780913524736)

found when remote player reading text pauses the game for everyone

nomaivr code [here](https://github.com/Raicuparta/nomai-vr/blob/master/NomaiVR/UI/LookArrow.cs#L138)

also apparently causes the freeze time text to come back? havent tested any of this

**Comment by thorio** [comment]
Seems plausible, we each had to disable time freezing from the main menu for it to work properly.

Without manually disabling, the host reading text would irrecoverably freeze the game for all players, requiring a force quit.
Not sure if it's just the host or clients as well.

Manually disabling all time freezing options fixed it, and the issue did not resurface in multiple hours of gameplay.

**Comment by JohnCorby** [comment]
there shouldnt be time freezing options in the first place

**Comment by JohnCorby** [comment]
oh nevermind we show the options on the title screen, just not in game

**Comment by JohnCorby** [comment]
potentially fixed?

**Comment by misternebula** [comment]
Yeah this should be fixed, if you could test it again that would be great!

**Comment by thorio** [comment]
So I finally got around to testing this.
I was able to reproduce the issue with the old version i had installed (one player in a multiplayer session), and after updating OWML and QSB, it went away.

So it is indeed fixed.

**Comment by misternebula** [comment]
nice, thanks for testing!
